tendermint/src/validator.rs

```diff
@@ -151,11 +131,12 @@ impl From<&Info> for InfoHashable {
     }
 }
 
-// returns the bytes to be hashed into the Merkle tree -
-// the leaves of the tree. this is an amino encoding of the
-// pubkey and voting power, so it includes the pubkey's amino prefix.
 impl Info {
-    fn hash_bytes(&self) -> Vec<u8> {
+    /// Returns the bytes to be hashed into the Merkle tree -
+    /// the leaves of the tree. this is an amino encoding of the
+    /// pubkey and voting power, so it includes the pubkey's amino prefix.
+    // TODO only used in lite impl ...
```

Comment:
Maybe we should remove this helper from this type if it is clear that it will only be used in the light client?

Reply:
Looking at `InfoHashable` and from what I remember from go tendermint it seems unlikely that this will be used not only in the light client.